src/utils/sortPeople.ts

```diff
@@ -0,0 +1,46 @@
+import { Person } from '../types';
+import { SortBy, SortOrder } from '../types/Sort';
+
+export function sortPeople(
+  people: Person[],
+  field: keyof Person,
+  order: string,
+): Person[] {
+  const sorted = [...people];
+
+  switch (order) {
+    case SortOrder.asc:
+      if (field === SortBy.name || field === SortBy.sex) {
+        sorted.sort((a, b) => {
+          return a[field].localeCompare(b[field]);
+        });
+      } else {
+        sorted.sort((a, b) => {
+          const n1 = a[field] ? +a[field] : 0;
+          const n2 = b[field] ? +b[field] : 0;
+
+          return n1 - n2;
+        });
+      }
+
+      break;
+
+    case SortOrder.decs:
```

Comment:
There seems to be a typo in the case condition. It should be `SortOrder.desc` instead of `SortOrder.decs`. This typo may cause the descending sort functionality to not work as expected.